set-web/src/main/java/eu/europeana/set/web/service/impl/BaseUserSetServiceImpl.java

```diff
@@ -488,6 +492,14 @@ void validateControlledValues(UserSet webUserSet) throws RequestBodyValidationEx
 	    throw new RequestBodyValidationException(UserSetI18nConstants.USERSET_VALIDATION_PROPERTY_VALUE,
 		    new String[] { WebUserSetModelFields.TYPE, webUserSet.getType() });
 	}
+	// if type is BookmarkFolder or Collection, validate subject
+	// subject must not contain entity reference
+	if(webUserSet.getType() != null && ! isEntityBestItemsSet(webUserSet)) {
+		if(webUserSet.isEntityBestItemSet()) {
```

Comment:
the method in the userset should be removed

Reply:
done